tests/search/e2e/support/commands.js

```diff
@@ -52,6 +52,7 @@ Cypress.Commands.add('visitAdminPage', (page = 'index.php') => {
 Cypress.Commands.add('openWidgetsPage', () => {
 	cy.login();
 	cy.visitAdminPage('widgets.php');
+	cy.wait(3000); // Wait for all DOM elements to be loaded
```

Comment:
I am surprised that Cypress does not have a function to wait for DOMContentLoaded :-(

Reply:
There is a way with chaining the `.should()` but that only works if that button element is there (which it may or may not be)